### PR TITLE
Frontend dev usersearch

### PR DESCRIPTION
**Pull request by sadullahgultekin**
I implemented search user page, can you check it out?

### CODE REVIEW

File: frontend/TraderX/src/views/search/index.vue [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      var temp = []
+      res.forEach(function (user) {
+        var privacy = user.isPrivate ? "Private" : !user.isPrivate ? 'Public': "";
+        temp.push({
+          'name': user.username,
+          'privacy' : privacy,
+          'role' : user.roles[0]
+        })
+      });
+      this.searchResult = deleteMultipleUsers(Array.from(new Set(temp)))
```
Comment (burakyuksel0) [inline frontend/TraderX/src/views/search/index.vue]: Why is this necessary? Is it about backend?

Comment (sadullahgultekin) [reply frontend/TraderX/src/views/search/index.vue]: Just as you said, it was unnecessary, I edit the file and deleted the unused lines
Thank you
